react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+let interval;
```

Comment:
We want to avoid global variables like this. If you need to store something that a class instance creates, you can stick it on `this`.

```js
this.interval = setInterval(() => {
```

Then you can access it later.
```js
clearInterval(this.interval)
```